networkx/algorithms/tree/branchings.py

```diff
@@ -709,68 +736,522 @@ def is_root(G, u, edgekeys):
 
 
 def maximum_branching(
+    G,
```

Comment:
For example, to make `G` position-only (i.e. users can't do `G=G`) and everything else keyword-only:
```suggestion
    G,
    /,
    *,
```

Reply:
I did this for the new function, `minimal_branching`, but wouldn't doing this to the other, older functions constitute an API change?

Reply:
Yup - sorry I meant to suggest this only for the new function :upside_down_face: